worlds/clique/__init__.py

```diff
@@ -32,78 +22,70 @@ class CliqueWorld(World):
     """The greatest game ever designed. Full of exciting gameplay!"""
 
     game = "Clique"
-    topology_present = False
-    data_version = 1
+    data_version = 2
     web = CliqueWebWorld()
     option_definitions = clique_options
 
-    location_name_to_id = location_table
-    item_name_to_id = item_table
-
-    def create_item(self, name: str) -> "Item":
-        return Item(name, ItemClassification.progression, self.item_name_to_id[name], self.player)
+    # Yes, I'm like 12 for this.
+    location_name_to_id = {
+        "The Big Red Button":   69696969,
+        "The Item on the Desk": 69696968,
+    }
 
-    def get_setting(self, name: str):
-        return getattr(self.multiworld, name)[self.player]
+    item_name_to_id = {
+        "Feeling of Satisfaction": 69696969,
+        "Button Activation":       69696968,
+    }
 
-    def fill_slot_data(self) -> dict:
-        return {option_name: self.get_setting(option_name).value for option_name in self.option_definitions}
+    def create_item(self, name: str) -> Item:
+        return Item(name, ItemClassification.progression, self.item_name_to_id[name], self.player)
```

Comment:
Do keep in mind that .game is not set like this, which makes it harder to figure out which game an item came from in debugging.

Reply:
Very fair. Was hoping to just remove the extra definitions, but probably best I don't. Will correct.